test/CardTests.cpp

```diff
@@ -0,0 +1,24 @@
+#include <gtest/gtest.h>
+#include "Card.hpp"
+#include <stdexcept>
+
+struct CardTests : public ::testing::Test
+{
+    Card redCard;
+};
+
+TEST_F(CardTests, check_if_call_setJob_out_of_range_will_throw_exception)
+{
+    ASSERT_THROW(redCard.setJob(-9, "Cofasz się o trzy pola." ), std::exception);
```

Comment:
If we don't care about the exception type we can use ASSERT_ANY_THROW. If we care then probably std::out_of_range is a proper exception type :)